reference/pipeline/templates/index.md

```diff
@@ -58,19 +58,20 @@ A pipeline template uses the following config JSON:
 A pipeline instance that implements a pipeline template uses the following
 config:
 
-```
+```json
 {
   "schema": "v2",
-  "application": "<appName>", # Set this to the app you want to create the pipeline in.
-  "name": "New Pipeline Name", # The name of your pipeline.
+  "application": "<appName>",     # Set this to the app you want to create the pipeline in.
+  "name": "New Pipeline Name",    # The name of your pipeline.
   "template": {
-    "source": "spinnaker://<pipelineTemplateName>" # The `id` field from the pipeline template.
-                                                   # Assuming the template was saved in Spinnaker,
-                                                   # you can prefix the id with ‘spinnaker://’.
-                                                   # ‘http://’ and ‘file://’ prefixes are also supported.
+    "type": "front50/pipelineTemplate",
+    "artifactAccount": "front50ArtifactCredentials",
+    "reference": "spinnaker://<templateName>"   # The `id` field from the pipeline template.
```

Comment:
Could you remove this comment? https://github.com/spinnaker/orca/pull/2667 removed the need for it.

Reply:
Thanks for the reference, I removed the comment.